netlify.toml

```diff
@@ -0,0 +1,19 @@
+[build]
```

Comment:
Just making sure this file addition was intentional

Reply:
Yes, this will enable the `version-2` branch to build deploy previews going forward (we can also remove the file if you'd rather it wasn't in there)!